src/phoenix/experimental/callbacks/llama_index_trace_callback_handler.py

```diff
@@ -330,3 +336,11 @@ def _message_payload_to_attributes(message: Any) -> Dict[str, Optional[str]]:
         MESSAGE_ROLE: "user",  # assume user if not ChatMessage
         MESSAGE_CONTENT: str(message),
     }
+
+
+def _message_payload_to_str(message: Any) -> Optional[str]:
+    """Converts a message payload to a string, if possible"""
+    if isinstance(message, ChatMessage):
+        return message.content
+
+    return str(message)
```

Comment:
I expect this might not work for the legacy completions endpoint. I can address as part of #1200 

https://github.com/jerryjliu/llama_index/pull/7160/files#diff-e3295fe390d39040f850d27cf3c0bc02959c54c0e9057e61686ff5dbec9c69b8R220-R223

Reply:
Yeah I'm not sure about all the cases but for the OpenAI agent right now this is a plain str.